cpp/src/py_main.cpp

```diff
@@ -628,7 +638,7 @@ PYBIND11_MODULE(libbtls, m) {
 			.def("get_no_modes", &CMultiModalNormal::getNoModes);
 	py::class_<CDistribution> cdistribution(m, "_Distribution");
 		cdistribution.def(py::init<>())
-			.def(py::init<double, double, double>(), py::arg("w"), py::arg("m"), py::arg("s"))
+			.def(py::init<double, double, double>(), py::arg("loc"), py::arg("scale"), py::arg("shape"))

```

Comment:
backwards compatible? Or doesn't affect the C++ code, just hte python argument names?

Reply:
The previous argument names were typos. Now they match the unchanged C++ code.